dev-libs/zziplib/zziplib-0.13.59.recipe

```diff
@@ -1,31 +1,94 @@
-DESCRIPTION="
-Lightweight library used to easily extract data from files archived in a \
-single zip file.
-"
+SUMMARY="The zziplib extract data from archives in a single zip"
+DESCRIPTION="Lightweight library used to easily extract data from \
+files archived in a single zip file."
 HOMEPAGE="http://zziplib.sourceforge.net/"
-SOURCE_URI="http://sourceforge.net/projects/zziplib/files/zziplib13/0.13.59/zziplib-0.13.59.tar.bz2/download"
+COPYRIGHT="1999-2010 Guido Draheim"
+LICENSE="GNU LGPL v2.1
+	MPL v1.1"
+SOURCE_URI="https://downloads.sf.net/zziplib/zziplib-$portVersion.tar.bz2"
 REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND=""
-CHECKSUM_MD5="14b5a6fc229afe9916d48358479568d3"
-BUILD()
+CHECKSUM_SHA256="cbc90cef60ee498319258e464895f990bb2e6e08f245264d77e44b309c8c218f"
+
+ARCHITECTURES="x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	zziplib$secondaryArchSuffix = $portVersion
+	cmd:zziplib$secondaryArchSuffix = $portVersion
+	cmd:unzip_mem
+	cmd:unzzip
+	cmd:unzzipcat
+	cmd:unzzipcat_mem
+	cmd:unzzipcat_seeko
+	cmd:unzzipdir
+ 	cmd:unzzipdir_mem
+	cmd:zzcat
+	cmd:zzdir
+	cmd:zzxorcat
+	cmd:zzxorcopy
+	cmd:zzxordir
+	lib:libzziplib$secondaryArchSuffix = $portVersion
+	lib:libzzip_0$secondaryArchSuffix  = 13.0.59 compat >=13.0.59
+	lib:libzzip$secondaryArchSuffix = $portVersion
+	lib:libzzipfseeko_0$secondaryArchSuffix = 13.0.59 compat >=13.0.59
+	lib:libzzipfseeko$secondaryArchSuffix = $portVersion
+	lib:libzzipmmapped_0$secondaryArchSuffix = 13.0.59 compat >=13.0.59
+	lib:libzzipmmapped$secondaryArchSuffix = $portVersion
+	lib:libzzipwrap_0$secondaryArchSuffix = 13.0.59 compat >=13.0.59
```

Comment:
This is what we get in the lib directory of the base package:
* libraries:
```
libzzip-0.so.13.0.59
libzzipfseeko-0.so.13.0.59
libzzipmmapped-0.so.13.0.59
libzzipwrap-0.so.13.0.59
```
* symbolic links:
```
libzzip-0.so.10 -> libzzip-0.so.13
libzzip-0.so.11 -> libzzip-0.so.13
libzzip-0.so.12 -> libzzip-0.so.13
libzzip-0.so.13 -> libzzip-0.so.13.0.59
libzzipfseeko-0.so.10 -> libzzipfseeko-0.so.13
libzzipfseeko-0.so.11 -> libzzipfseeko-0.so.13
libzzipfseeko-0.so.12 -> libzzipfseeko-0.so.13
libzzipfseeko-0.so.13 -> libzzipfseeko-0.so.13.0.59
libzzipmmapped-0.so.10 -> libzzipmmapped-0.so.13
libzzipmmapped-0.so.11 -> libzzipmmapped-0.so.13
libzzipmmapped-0.so.12 -> libzzipmmapped-0.so.13
libzzipmmapped-0.so.13 -> libzzipmmapped-0.so.13.0.59
libzzipwrap-0.so.13 -> libzzipwrap-0.so.13.0.59
```
So I think we should have these lines in PROVIDES:
```
	lib:libzzip_0$secondaryArchSuffix = $portVersion compat >= 10
	lib:libzzipfseeko_0$secondaryArchSuffix = $portVersion compat >= 10
	lib:libzzipmmapped_0$secondaryArchSuffix = $portVersion compat >= 10
	lib:libzzipwrap_0$secondaryArchSuffix = $portVersion compat >= 13
```
But I'm not sure if HaikuPorter will accept the ">= 10" because "13.0.59" does not start with "10".